src/main/java/se/sundsvall/garbage/service/scheduler/UpdateGarbageSchedulesScheduler.java

```diff
@@ -1,23 +1,31 @@
 package se.sundsvall.garbage.service.scheduler;
 
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.scheduling.annotation.Scheduled;
 import org.springframework.stereotype.Service;
 
-import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
 import se.sundsvall.garbage.service.GarbageService;
 
+import net.javacrumbs.shedlock.spring.annotation.SchedulerLock;
+
 @Service
 public class UpdateGarbageSchedulesScheduler {
 
 	private final GarbageService garbageService;
 
-	public UpdateGarbageSchedulesScheduler(GarbageService garbageService) {
+	@Value("#{'${schedulers.update-garbage-schedules.municipality-ids}'.split(',')}")
```

Comment:
```suggestion
	@Value(${schedulers.update-garbage-schedules.municipality-ids:})
```

This should work just fine, no need to explicitly split it like you have. (":" is used to set a default value, so in case the property is not set, it will be an empty list)